bundles/binding/org.openhab.binding.maxcube/src/main/java/org/openhab/binding/maxcube/internal/MaxCubeBinding.java

```diff
@@ -186,10 +199,14 @@ public synchronized void execute() {
 					cont = false;
 					continue;
 				}
-
-				Message message;
+				
+				Message message = null;
 				try {
-					message = processRawMessage(raw);
```

Comment:
I think this needs to be consistent. If we replace message = processRawMessage(raw); then replace it in the send message area (L435) as well